app/realtime.php

```diff
@@ -373,25 +301,24 @@ function getCache(): Cache
 
                     if ($realtime->hasSubscriber($projectId, 'user:' . $userId)) {
                         $connection = array_key_first(reset($realtime->subscriptions[$projectId]['user:' . $userId]));
-                        $consoleDatabase = getConsoleDB();
-                        $project = Authorization::skip(fn () => $consoleDatabase->getDocument('projects', $projectId));
-                        $database = getProjectDB($project);
+                        $consoleDatabase = $container->get('dbForConsole');
```

Comment:
```suggestion
                        $dbForConsole = $container->get('dbForConsole');
```
lets call it this way